README.md

```diff
@@ -103,6 +103,41 @@ BarPageState extends State<BarPage>(
 )
 ```
 
+### Controller
+
+The wizard can be accessed from anywhere in the widget tree by using a `WizardController`.
+
+```dart
+class FooPage extends StatelessWidget {
```

Comment:
could we make it a stateful widget and dispose the controller to give a more correct starting point example?

Reply:
"More" correct, meaning just "correct" :laughing: